include/mata/nfa/nfa.hh

```diff
@@ -267,24 +268,6 @@ public:
      */
     std::vector<Trans> get_trans_from_as_sequence(State state_from) const;
```

Comment:
Transitions.to_vector() 

or can we remove it and just call vector<Trans>(Transitions.betin(),Transitions.end()) ?

Reply:
Now that we have iterators over transitions of type `Transition`, this function can be removed entirely.

Reply:
Replaced with iterator over transitions `Delta::Transitions::from(source)`.